games-vue-client/src/components/StartScreen.vue

```diff
@@ -1,8 +1,21 @@
 <template>
   <v-container fluid>
   <v-row>
+    <v-col cols="12" md="6" lg="4">
+      <v-row>
+        <v-col cols="2" md="2" lg="2">
```

Comment:
Is it necessary to have a v-col in v-row in a v-col in a v-row here? Wouldn't one v-row and the two innermost v-col be enough? Would it be a difference in the view of different devices?

Reply:
It's not necessary to wrap like I did here, but I like the system of having two or three main columns and arrange items in these main columns into smaller pieces. That gives more control over the white space.

If it's too much for the moment then I will try another solution.